lms/djangoapps/courseware/management/commands/dump_to_neo4j.py

```diff
@@ -0,0 +1,184 @@
+"""
+This file contains a management command for exporting the modulestore to
+neo4j, a graph database.
+"""
+import logging
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from py2neo import Graph, Node, Relationship
+from py2neo.compat import integer, string, unicode as neo4j_unicode
+from request_cache.middleware import RequestCache
+from xmodule.modulestore.django import modulestore
+
+log = logging.getLogger(__name__)
+
+bolt_log = logging.getLogger('neo4j.bolt')  # pylint: disable=invalid-name
```

Comment:
nit: I assume this is in py2neo? Should you point out it's a custom logging scheme for py2neo?